pkg/knuu/instance.go

```diff
@@ -242,9 +242,9 @@ func (i *Instance) ExecuteCommand(command ...string) (string, error) {
 	return "", nil
 }
 
-// AddFile adds a file to the instance
+// Add adds to the instance
 // This function can only be called in the state 'Preparing'
-func (i *Instance) AddFile(src string, dest string, chown string) error {
+func (i *Instance) Add(src string, dest string, chown string) error {
```

Comment:
Currently, I'm not sure if I like the function name `Add()`. Because from an Instance view, what does `add` mean? An Instance abstracts Docker and Kubernetes, and it's unclear that `add` means adding a file/directory.

Reply:
Also, I created https://github.com/celestiaorg/knuu/pull/77 to resolve https://github.com/celestiaorg/knuu/issues/66, but I forgot to close the issue. I will keep this PR open for now and decide together with @jrmanes how we want to move forward regarding this topic.